### PR TITLE
ENDOC-456 updated images and doc for assets tutorial

### PR DESCRIPTION
**Pull request by jyunmitch**

### CODE REVIEW

File: vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md [hunk 1]
```diff
@@ -1,136 +1,101 @@
...
+1. Go to `Content` → `Assets` → `Add` 
+2. After uploading an image, click on the actions button to the right of the image. From the drop-down menu, Select Edit.
+![Add Digital Assets Window](./img/assets-edit-image.png)
+3. Use the arrows to flip, rotate or recenter your image. When done, click the `Crop` check button. Multiple resized images can be created at this time. Click `Free` to remove resizing ratio restrictions.
+4. Click `Done`.
+5. Click Add to save all versions.
 
-![image](./img/cms_digital_assets_crop.png)
+## Embed an Asset on a Page
+The uploaded asset can be embedded on a page by using `<@wp.resourceUrl>your-path/your-asset.jpg`. The path can be found in the App Builder at `Administration` → `File Browser`. (e.g.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md:59]: @nshaw should there be a space between '>' and 'your-path/...'?
<@wp.resourceUrl>your-path/your-asset.jpg

File: vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md [hunk 2]
```diff
@@ -1,136 +1,101 @@
...
+1. From the Content Assets page in the App Builder, click `Add` on the right to add a new image or document file. 
 
--   **Documents**: All added documents are displayed. 
+![Add Digital Assets Window](./img/assets-add.png)
 
-## Editing Assets
-Once you have a digital asset in place, you have several editing and viewing options. Click the asset's *Actions* button and select **Edit** to update the asset's name/description, categories, and the file represented.
+2. Group: choose from the drop-down menu. This is a required field and defaults to `Free Access`. Additional groups can be defined in the Users section.  
+3. Categories: add your asset to the appropriate category with the `+` symbol under `Join`. This will be used in searching and sorting. This field is not mandatory. New Categories can be created in `Content` → `Categories`.
+4. Drag and Drop your files into the window or click `Browse`. The file you choose will appear in the Name field. You can choose multiple files at one time. You can also use the `Add Another Resource` button and `Choose file` individually.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md:36]: "will appear" -> "appears" (present tense)

"`Choose file` individually" isn't grammatically correct... i know you're trying to avoid redundancy here... "For individual selections you can use the `Add Another Resource` button and `Choose file`."?

File: vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md [hunk 3]
```diff
@@ -1,136 +1,101 @@
...
+5. After all files have been chosen, click the `Add` button to upload the files. This takes you back to the main assets page where your uploaded files will be displayed in the table. If you have an image file, see Edit Assets below.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md:37]: " are displayed" (present tense)

maybe "...Edit Assets section" just to be clear?

File: vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md [hunk 4]
```diff
@@ -1,136 +1,101 @@
...
+2. Group: choose from the drop-down menu. This is a required field and defaults to `Free Access`. Additional groups can be defined in the Users section.  
+3. Categories: add your asset to the appropriate category with the `+` symbol under `Join`. This will be used in searching and sorting. This field is not mandatory. New Categories can be created in `Content` → `Categories`.
+4. Drag and Drop your files into the window or click `Browse`. The file you choose will appear in the Name field. You can choose multiple files at one time. You can also use the `Add Another Resource` button and `Choose file` individually.
+5. After all files have been chosen, click the `Add` button to upload the files. This takes you back to the main assets page where your uploaded files will be displayed in the table. If you have an image file, see Edit Assets below. 
+6. If a Name field is left blank, you will receive an error message. Simply go to the actions (indicated by three vertical dots) menu next to the empty file and select Delete. This action menu also provides an edit option for each resource.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md:38]: "the Actions" and "the Actions menu" ??

File: vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md [hunk 5]
```diff
@@ -1,136 +1,101 @@
...
 
-### Cropping Images
-You can crop images newly uploaded via Digital Assets. You can either crop the image dynamically or by using pre-configured crop ratios.
+## Edit Assets 
+Changes to the Asset name and categories can be made in the Edit mode.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md:41]: "in Edit mode" -> no "the"

File: vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md [hunk 6]
```diff
@@ -1,136 +1,101 @@
...
+4. Drag and Drop your files into the window or click `Browse`. The file you choose will appear in the Name field. You can choose multiple files at one time. You can also use the `Add Another Resource` button and `Choose file` individually.
+5. After all files have been chosen, click the `Add` button to upload the files. This takes you back to the main assets page where your uploaded files will be displayed in the table. If you have an image file, see Edit Assets below. 
+6. If a Name field is left blank, you will receive an error message. Simply go to the actions (indicated by three vertical dots) menu next to the empty file and select Delete. This action menu also provides an edit option for each resource.
 
-### Cropping Images
-You can crop images newly uploaded via Digital Assets. You can either crop the image dynamically or by using pre-configured crop ratios.
+## Edit Assets 
+Changes to the Asset name and categories can be made in the Edit mode.  
+1. From `Content` → `Assets` page, in the list of assets, click on the Asset's `Actions` button and select `Edit`. 
+2. Edit the Name field. For Categories, use the `+` button under `Join` to add the asset to a category.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md:43]: capitalize "asset"? is capitalized in step above

can simplify second sentence in step 2: "Use the `+` button under `Join` to add the Asset to a category." ?

File: vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md [hunk 7]
```diff
@@ -1,136 +1,101 @@
...
+`<img src="<@wp.resourceUrl> cms/images/butterfly.jpg">`
 
-- Use your mouse or the buttons below the image to move or change the crop area, then click *Save* to crop the selected area. Repeat the process as many times as needed.
-  - Click *Close* to return to the Digital Assets screen.
+![File Browser](./img/assets-filebrowser.png)
 
-## Embedding an Asset in a Page
-The uploaded asset can be embedded in a page by using `<@wp.resourceUrl path/to/asset`. The path can be found in the CMS Configuration > File Browser page.
+## Include an Asset in a Content
+The asset can be included in a Content Type that has an `Attach` or `Image` attributes. Follow the [Content Type tutorial](./content-types-tutorial.md) for detailed steps or follow the briefly outlined steps here.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md:65]: "attribute" (not plural)

"the asset" -> capitalize "asset" because referring to the one you're working with?

File: vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md [hunk 8]
```diff
@@ -1,136 +1,101 @@
...
 
-![image](./img/cms_file_browser.png)
+Create a Content Type with an image attribute. 
+1. From the left sidebar select `Content` → `Types` → `Add`
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md:68]: comma after "sidebar"?

only step in this section that doesn't end with a period

File: vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md [hunk 9]
```diff
@@ -1,136 +1,101 @@
...
 
-Example usage:
-`<img src="<@wp.resourceUrl cms/images/butterfly_d0_d3.jpg"`
+![Create Content Type](./img/assets-add-type.png)
 
-## Including an Asset in a Content
-The asset can be included in a content of a type that has an *Attach* or *Image* attribute.
+2. Fill out `Code`, `Name` and click `Add`.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md:72]: "...`Code` and `Name`, then..." a little cleaner

File: vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md [hunk 10]
```diff
@@ -1,136 +1,101 @@
...
+1. From the left sidebar select `Content` → `Types` → `Add`
 
-Example usage:
-`<img src="<@wp.resourceUrl cms/images/butterfly_d0_d3.jpg"`
+![Create Content Type](./img/assets-add-type.png)
 
-## Including an Asset in a Content
-The asset can be included in a content of a type that has an *Attach* or *Image* attribute.
+2. Fill out `Code`, `Name` and click `Add`.  
+3. This will take you to the Add Attribute page. In this example, we are using the Attribute Type of image. Enter the `Code` and `Name` as required.  All other fields are optional. Click `Continue` at bottom of page. Note the Attribute Code field is used below to reference the image in the Template.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md:73]: "takes you" -> present tense

"the Attribute Type of image" reads like there are other types of images... would rephrase to "...`image` as the Attribute Type."

"as required" implies that sometimes this is not required. would rephrase to "It is mandatory to enter values for `Code` and `Name` but all other fields are optional."

"of the page"

File: vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md [hunk 11]
```diff
@@ -1,136 +1,101 @@
...
+![image](./img/assets-add-attribute.png)
+4. Next create the Template or Model where the content type is added. Go to `Content` → `Template` → `Add`
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md:75]: can omit "Next"

content type -> Content Type

create a template/model where the type is added reads like you create these in the neighborhood of where you see an add option for Content Type

File: vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md [hunk 12]
```diff
@@ -1,136 +1,101 @@
...
+![image](./img/assets-template.png)
+5. From the drop-down menu for `Type`, select the one you just created.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md:77]: reads a little haltingly; a little cleaner to switch first and last part of sentence, e.g. "Select the `Type` you just created from the drop-down menu."

File: vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md [hunk 13]
```diff
@@ -1,136 +1,101 @@
...
+6. Enter the `Code` and `Name` fields as required.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md:78]: can nix "as required"

File: vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md [hunk 14]
```diff
@@ -1,136 +1,101 @@
...
-## Including an Asset in a Content
-The asset can be included in a content of a type that has an *Attach* or *Image* attribute.
+2. Fill out `Code`, `Name` and click `Add`.  
+3. This will take you to the Add Attribute page. In this example, we are using the Attribute Type of image. Enter the `Code` and `Name` as required.  All other fields are optional. Click `Continue` at bottom of page. Note the Attribute Code field is used below to reference the image in the Template.
+![image](./img/assets-add-attribute.png)
+4. Next create the Template or Model where the content type is added. Go to `Content` → `Template` → `Add`
+![image](./img/assets-template.png)
+5. From the drop-down menu for `Type`, select the one you just created.
+6. Enter the `Code` and `Name` fields as required. 
+7. Complete the Model HTML layout in Velocity. The Inline Editing Assist tool is available by using the Ctl-Space buttons on your keyboard. Your image tag should look something like this:
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md:79]: by using -> by pressing

File: vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md [hunk 15]
```diff
@@ -1,136 +1,101 @@
...
+## Configure File Extensions for Upload
 
-The types of files that can be uploaded to an Entando application are controlled by
-configuration in the server side of the app builder. If you're using the quickstart this will be the
-`entando-de-app`.
+The types of files that can be uploaded to an Entando Application are controlled by the
+configuration of the server side of the App Builder. If you're using the quickstart version, this is the`entando-de-app`.
 
-There are two properties that control these settings and they are set in `src/main/conf/systemParams.properties` and are a comma delimitted list of values. The property keys are
+There are two properties that control these settings and they are set in `src/main/conf/systemParams.properties` and are a comma delimitted list of values. The property keys are:
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md:89]: run-on sentence. would split in two -> "...settings. They are..."

File: vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md [hunk 16]
```diff
@@ -1,136 +1,101 @@
...
 
-- `jacms.imageResource.allowedExtensions=`
-- `jacms.attachResource.allowedExtensions=`
+* `jacms.imageResource.allowedExtensions=`
+* `jacms.attachResource.allowedExtensions=`
 
-If you are configuring the upload for both the App Builder and the legacy Admin Console you should include
-both the MIME type of the asset and the file extension. The APIs that support resources will check MIME type of the uploaded asset.
+To configure the uploads for the App Builder and Admin Console, include both the MIME type and file extension of the asset. The APIs that support resources will check the MIME type of the uploaded asset.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md:94]: "the file extension"

"will check" -> "check" (present tense)

File: vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md [hunk 17]
```diff
@@ -1,136 +1,101 @@
...
 
-An example configuration for images.
+An example configuration for images:
 - `jacms.imageResource.allowedExtensions=jpg,jpeg,png,svg,svg+xml`
 
 Where `svg+xml` is the MIME type for an svg image.
 
-The `systemParams.properties` file is bundled into the WAR and the image for the application so you will need to create and deploy image to reflect changes for these properties in your application.
+The `systemParams.properties` file is bundled into the WAR and the image for the application, so you will need to create and deploy an image to reflect changes for these properties in your application.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md:101]: what is a WAR? should spell out acronym?

"you need to" -> no "will" (present tense)

"to" these properties, not "for"

File: vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md [hunk 18]
```diff
@@ -1,136 +1,101 @@
...
-![image](./img/cms_content_image_add.png)
 
-## Configuring Allowed File Extensions for Upload
+## Configure File Extensions for Upload
 
-The types of files that can be uploaded to an Entando application are controlled by
-configuration in the server side of the app builder. If you're using the quickstart this will be the
-`entando-de-app`.
+The types of files that can be uploaded to an Entando Application are controlled by the
+configuration of the server side of the App Builder. If you're using the quickstart version, this is the`entando-de-app`.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md:87]: no "the" before `entando...`

File: vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md (hunk 1 above)
Comment (jyunmitch) [reply vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md:59]: another citation in the doc had a space so checking

**Review by Lyd1aCla1r3 — APPROVED** [review]
